cfgov/unprocessed/css/pages/ask.less

```diff
@@ -105,6 +104,99 @@
 /* Answer page */
 
 .ask-cfpb-page__answer {
+    .respond-to-min( @bp-lg-min, {
+        .content_sidebar {
+            padding-top:0;
+        }
+    });
+
+    .o-expandable_cue-close {
+        display: none;
+    }
+
+    .answer-text {
+        h2,
+        h3,
+        h4 {
+            margin-top: 30px;
```

Comment:
Mix of hardcoded margins and `@grid_gutter-width` here and throughout. Looks like this should be `margin-top: @grid_gutter-width;`.

Reply:
Should be fixed with latest commits.